tools/accuracy_checker/accuracy_checker/metrics/attribute_classification.py

```diff
@@ -0,0 +1,154 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from sklearn.metrics import accuracy_score, precision_score, recall_score
```

Comment:
in some platform like yocto, sklearn may be unavailable, so you metric can not work for it. If it is not important for you, then stay it as it, otherwise please rewrite without sklearn usage.